css/src/components/form/help.scss

```diff
@@ -0,0 +1,24 @@
+$help-color: $text !default;
+$help-error-color: $danger !default;
+$help-warning-color: $warning-active !default;
+$help-success-color: $score-high !default;
+$help-font-size: $font-size-8 !default;
+
+.help {
+	display: block;
+	margin-block-start: 0.5rem;
```

Comment:
I don't think we should include any margin on elements by default. We can use atomics to compose experiences.

Reply:
I think we should provide some default spacing on it. It is meant to be used in combination with some other component, so having some spacing would be needed always. What's the point of figuring out later what additional atomics to use when we can prevent it? 🤔

Reply:
A general principal of all our components is that they have no margin sizing by themselves. This allows for easy composability in a variety of scenarios.

Reply:
Removed